modules/virt-about-pci-passthrough.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-pci-passthrough.adoc
+
+[id="virt-about_pci-passthrough_{context}"]
+= About PCI passthrough
+
+The Peripheral Component Interconnect (PCI) passthrough feature enables you to access and manage hardware devices from a virtual machine. When PCI Passthrough is configured, the PCI devices function as if they were physically attached to the virtual machine.
```

Comment:
The second sentence is still capitalizing "Passthrough" but I think you have it lowercase elsewhere.

Reply:
Lowercased the word as passthrough.